frontend/device.lua

```diff
@@ -32,7 +32,7 @@ local function probeDevice()
     --     return require("device/newport/device")
     -- end
 
-    error("did not find a hardware abstraction for this platform")
+    error("did not find a hardware abstraction for this platform, maybe you need to install SDL if you're on Linux?")
```

Comment:
Except for Linux But Not Really™ Android they're definitely all Linux. :-P

I'd say something like
```
Could not find hardware abstraction for this platform. If you are trying to run the emulator, please ensure SDL is installed.
```